convert_torcx_image.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+set -euo pipefail
+
+if [ $# -lt 2 ] || [ "$1" = "-h" ] || [ "$1" = "--help" ]; then
+  echo "Usage: $0 TORCXTAR SYSEXTNAME"
+  echo "The script will unpack the Torcx tar ball and create a sysext squashfs image with the name SYSEXTNAME.raw in the current folder."
+  echo "A temporary directory named SYSEXTNAME in the current folder will be created and deleted again."
+  echo "All files in the sysext image will be owned by root."
+  echo "The conversion is done on best effort, and only the TORCX_BINDIR (/usr/bin), TORCX_UNPACKDIR (/), and TORCX_IMAGEDIR ('') env vars in the systemd units get replaced."
+  exit 1
+fi
+
+TORCXTAR="$1"
+SYSEXTNAME="$2"
+
+rm -rf "${SYSEXTNAME}"
+mkdir -p "${SYSEXTNAME}"
+tar -xf "${TORCXTAR}" -C "${SYSEXTNAME}"
+rm -rf "${SYSEXTNAME}"/.torcx
+mkdir -p "${SYSEXTNAME}"/usr
+if [ -e "${SYSEXTNAME}"/bin ]; then
+  mv "${SYSEXTNAME}"/bin "${SYSEXTNAME}"/usr/
+fi
+if [ -e "${SYSEXTNAME}"/lib ]; then
+  mv "${SYSEXTNAME}"/lib "${SYSEXTNAME}"/usr/
+fi
+for FILE in "${SYSEXTNAME}"/usr/lib/systemd/system/*.service "${SYSEXTNAME}"/usr/lib/systemd/system/*.socket; do
+  if [ -e "${FILE}" ]; then
+    sed -i -e 's,${TORCX_BINDIR},/usr/bin,g' -e 's,${TORCX_UNPACKDIR},/,g' -e 's,${TORCX_IMAGEDIR},,g' \
```

Comment:
What do you think of overwriting the environment with `Environment=TORCX_BINDIR=/usr/bin`, etc. instead? Or overwriting `EnvironmentFile` (is that possible?) to a file contained within the sysext. That way drop-ins continue working? Or is this carrying too much legacy with us.

Reply:
The problem with the drop-ins is that they stop working as soon as one would switch to the inbuilt sysext image. We could start to encode these variables there, too, though…?

Reply:
Ok, updated it now to set the environment